content/shared/tutorials/_testing_push_ios.textile

```diff
@@ -0,0 +1,103 @@
+  var notification = {
+    notification: {
+      title: 'Hello from Ably!'
+    }
+  };
+  realtime.push.publish(recipient, notification, function(err) {
+    if (err) {
+      console.log('Unable to publish push notification; err = ' + err.message);
+      return;
+    }
+    console.log('Push notification published');
+    res.send("Push Sent");
+  });
+})
+```
+
+Restart the server and navigate to http://localhost:3000/push/device in your browser. You should be able to receive a notification on your device as shown the image below.
+
+<a href="/images/tutorials/push-ios/apple-push-ios-title.png" target="_blank">
+	 <img src="/images/tutorials/push-ios/apple-push-ios-title.png" style="width: 100%" alt="Apple Push With Title">
+</a>
+
+h3. Testing using Client ID
+
+We can test sending push notifications using the client ID, this allows us to push all devices with that client ID by adding the following code to your index.js file:
+
+```[nodejs]
+app.get('/push/client', function (req, res) {
+  var recipient = {
+    clientId: 'iosPushClient'
+  };
+  var notification = {
+    notification: {
+      title: 'Hello from Ably!'
+    }
+  };
+  realtime.push.publish(recipient, notification, function(err) {
+    if (err) {
+      console.log('Unable to publish push notification; err = ' + err.message);
+      return;
+    }
+    console.log('Push notification published');
+    res.send("Push Sent");
+  });
+})
+```
+Note: If your clientId is different you will also need to change this.
+
+Restart the server and go to http://localhost:3000/push/client you should receive a notification on your device as shown in the image below.
+
+<a href="/images/tutorials/push-ios/apple-push-ios-title.png" target="_blank">
+	 <img src="/images/tutorials/push-ios/apple-push-ios-title.png" style="width: 100%" alt="Apple Push With Title">
+</a>
+
+h3. Testing using Channels
+
+Channel based Push Notifications allow us to push messages to a channel that the client on the device has subscribed to. To test this, add the following code to our index.js:
+
+```[nodejs]
+app.get('/push/channel', function (req, res) {
+  var extras = {
+    push: {
+      notification: {
+        title: 'Hello from Ably!',
+        body: 'Example push notification from Ably.'
+      },
+      data: {
+        foo: 'bar',
+        baz: 'qux'
+      }
+    }
+  };
+
+  var channel = realtime.channels.get('push');
+  channel.publish({ name: 'example', data: 'data', extras: extras }, function(err) {
+    if (err) {
+      console.log('Unable to publish message with push notification; err = ' + err.message);
+      return;
+    }
+    console.log('Message with push notification published');
+    res.send("Push Sent");
+  });
+})
+```
+
+Restart the server and go to http://localhost:3000/push/channel, you should receive a notification on your device. Notice how this notification has both a title and a body. You can do this on all three types of push notifications mentioned earlier, if you wish.
```

Comment:
@cjbatin you've mentioned that users can implement the visual part in the previous types as well, please explain what's causing the title and the body to show up in the third example - that they could include in the previous ones as well.